server/data/quotes.json

```diff
@@ -271,6 +271,10 @@
     "author": "Jules Michelet",
     "sentence": "The most intimate temper of a people, its deepest soul, is above all in its language."
   },
+  {
+    "author": "Swami Vivekananda",
```

Comment:
Please add it alphabetically under the quotes starting with the author name of "S".